src/resonanceReconstruction/reichMoore/Type/test/Type.test.cpp

```diff
@@ -32,6 +32,7 @@ SCENARIO( "Integration test" ){
   SECTION( "Iron-56" ){
     const auto Fe56 = resonances("Fe-56");
 
+    auto start = std::chrono::high_resolution_clock::now();
```

Comment:
you're conflating the parsing step and reconstruction step by beginning your timer here. If you're interested in comparing to `csrmat`, you should move this to (what is now) line 50